templates/puppet.yaml.erb

```
@@ -12,4 +12,5 @@
 :timeout: <%= @enc_timeout %>
 :report_timeout: <%= @reports_timeout %>
 :report_retry_limit: <%= @report_retry_limit %>
+:report_ignore_facter_errors: "<%= @report_ignore_facter_errors %>"
```

Booleans shouldn't be quoted
```suggestion
:report_ignore_facter_errors: <%= @report_ignore_facter_errors %>
```